app/javascript/graph_builder.js

```diff
@@ -99,3 +100,16 @@ document.addEventListener('click', event => {
 
   segChart.reflow()
 })
+
+window.addEventListener('resize', _.debounce(() => {
+  Highcharts.charts.forEach((chart) => {
+    const row = chart.renderTo.closest('tr')
+    if (row === null) {
+      return
+    }
+
+    _.defer((size) => {
+      chart.setSize(size)
+    }, chart.container.closest('table').closest('.card').clientWidth)
```

Comment:
Can you also include the reason for this defer being necessary in a comment here? Since it's not obvious looking at the code on its own.